src/js/components/ThumbsRating/ThumbsRating.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Like, LikeFill, Dislike, DislikeFill } from 'grommet-icons';
+import { ThumbsRatingPropTypes } from './propTypes';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const ThumbsRating = ({ color, options, ...rest }) => {
```

Comment:
Similar question about `color` in the first version as with star rating.

Reply:
we could leave out designs for hpe use `color` are you opposing it as a prop and maybe have it in theme ?

Reply:
yes

Reply:
sounds good I can adjust this 👍